ui/widgets/tappablewrappers.go

```diff
@@ -81,6 +81,13 @@ func (t *TappableImage) Refresh() {
 	t.BaseWidget.Refresh()
 }
 
+func (t *TappableImage) MinSize() fyne.Size {
+	if !t.haveImage() {
```

Comment:
you can just return t.Image.MinSize() here since even if it doesn't have an `image.Image` assigned to it, the `canvas.Image` still has a minsize that can be set by wherever we're using the tappable image.

This TappableImage is a little weird I think because it embeds both widget.BaseWidget and canvas.Image. Normally for extending a widget you don't have to re-define all of the interface methods, but in this case I had to.

Reply:
Good call! Change applied.